src/transformers/onnx/convert.py

```diff
@@ -62,11 +62,15 @@ def check_onnxruntime_requirements(minimum_version: Version):
         )
 
 
-def export(
-    tokenizer: PreTrainedTokenizer, model: PreTrainedModel, config: OnnxConfig, opset: int, output: Path
+def export_pytorch(
```

Comment:
Note: we decided to refactor the single `export()` function into dedicated functions for PyTorch and TensorFlow